graphql_query_derive/src/lib.rs

```diff
@@ -15,35 +15,28 @@ use proc_macro2::TokenStream;
 pub fn derive_graphql_query(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
     match graphql_query_derive_inner(input) {
         Ok(ts) => ts,
-        Err(err) => panic!(
-            "{}",
-            err.iter_chain()
-                .fold(String::new(), |mut acc, item| {
-                    acc.push_str(&format!("{}\n", item));
-                    acc
-                })
-                .trim_end_matches('\n')
-        ),
+        Err(err) => panic!("{:?}", err,),
```

Comment:
Trailing comma looks weird without being splayed out.

```suggestion
        Err(err) => panic!("{:?}", err),
```